doc/concepts/configuration/configuration_code.rst

```diff
@@ -0,0 +1,341 @@
+..  _configuration_code:
+
+Configuration in code
+=====================
+
+.. box_cfg_legacy_note_start
+
+.. NOTE::
+
+    Starting with the 3.0 version, the recommended way of configuring Tarantool is using a :ref:`configuration file <configuration_file>`.
+    Configuring Tarantool in code is considered a legacy approach.
+
+.. box_cfg_legacy_note_end
+
+This topic covers specifics of configuring Tarantool in code using the ``box.cfg`` API.
+In this case, a configuration is stored in an :ref:`initialization file <index-init_label>` - a Lua script with the specified configuration options.
+You can find all the available options in the :ref:`Configuration reference <box-cfg-params-ref>`.
+
+
+..  _index-init_label:
+
+Initialization file
+-------------------
+
+If the command to :ref:`start Tarantool <configuration_code_run_instance_tarantool>` includes ``LUA_INITIALIZATION_FILE``, then
+Tarantool begins by invoking the Lua program in the file, which by convention
+may have the name ``script.lua``. The Lua program may get further arguments
```

Comment:
`LUA_INITIALIZATION_FILE` is used as a term, while we have no specific term for the instance file. One may assume that there is a same named environment variable or other configuration facility that is related to the non-existing term.

`script.lua` is used like it is a usual name for the instance file, while it isn't. Several years ago there were no usual instance file name -- it is called according to a microservice name. Now, many applications that are based on cartridge framework use `init.lua`. It is not necessary, though.

I have two options in the mind. First, call it 'instance file'.

Second, if it is easier to write the documentation using certain name, add a remark.

> While the instance file name is arbitrary, this article uses `init.lua` file name for simplicity.

Or something of this kind.

Reply:
Thanks, will apply your suggestions